protocol-definitions/VectorCollection.yaml

```diff
@@ -272,3 +272,19 @@ methods:
             A null value triggers the optimization of the only index within the collection.
     response: {}
 
+  - id: 10
+    name: clear
+    since: 2.8
+    doc: |
+      Clear vector collection.
+    request:
+      retryable: true
```

Comment:
I wonder if it should be retryable. IMap.clear is not

Reply:
As a mutating operation, it's safer to go with `false`

Reply:
03064e0df406a2e64f6b378874f28fece5a9da18